tests/test_blobstore.py

```diff
@@ -35,70 +36,80 @@ def test_schema(self):
         self.assertEqual("gs://", GSBlobStore.schema)
 
     def test_get(self):
-        with self.subTest("aws"):
-            key = f"{uuid4()}"
-            expected_data = test_data.oneshot
-            self._put_s3_obj(s3_test_bucket, key, expected_data)
-            data = S3BlobStore(s3_test_bucket).blob(key).get()
-            self.assertEqual(data, expected_data)
-        with self.subTest("gcp"):
-            key = f"{uuid4()}"
-            expected_data = test_data.oneshot
-            self._put_gs_obj(s3_test_bucket, key, expected_data)
-            data = GSBlobStore(gs_test_bucket).blob(key).get()
-            self.assertEqual(data, expected_data)
+        key = f"{uuid4()}"
+        expected_data = test_data.oneshot
+        tests = [("aws", self._put_s3_obj, s3_test_bucket, S3BlobStore),
```

Comment:
Is this list something that could be in the setupClass and called for tests as `self.tests`?

Reply:
That'd be nice. Not all of the tests have the same parameters, but perhaps they overlap enough to make a class method useful.

Something for an intern to look into :)